src/Extension/ValueResolver/ModuleValueResolver.php

```diff
@@ -1,90 +0,0 @@
-<?php
```

Comment:
Please, mark as deprecated instead of removing it. We'll remove it in the next major release.